Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/FilterAdapter.h

```diff
@@ -21,6 +21,10 @@ namespace AZ::DocumentPropertyEditor
 
         void SetIncludeAllMatchDescendants(bool includeAll);
 
+        // MetaAdapter overrides
```

Comment:
uber nit: For consistency missing the ellipsis at the end (`// MetaAdapter overrides ...`)

Reply:
fixed? Well, at least changed now :P